### PR TITLE
Additional shoot buttons

### PR DESCRIPTION
**Pull request by cbf9427**
Add four new shoot buttons using the Orange 4,5,9,10 set on the Farm controller.

4 - same as current shot
5 - about 2.5' further out
9 - near Launchpad shot
10 - far Launchpad shot

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -88,9 +88,30 @@
...
+    // Various constants used for shooting
+    //
 
-    // chute subsystem
+    // The shooting mechanism is about 24" from the edge of the robot, including bumper
+    // We therefore subtract SHOOTER_POS from all shooting distance constants
+    public static final double SHOOTER_POS = 24;
+    
+    // Our usual "Tarmac" shot is 9ft
+    public static final double TARMAC_DEFAULT_DISTANCE = (12.0 * 9.0) - SHOOTER_POS;
```
Comment (prensing) [inline src/main/java/frc/robot/Constants.java:100]: This was tweaked at Revere to be a bit longer, 94 in. Should be set back to that value.

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -185,8 +206,8 @@
     public static final double CLIMB_TO_NEXT_BAR_TIMEOUT = 12.0;
 
     // robot starting distance
-    public static final double STARTING_DISTANCE = 74.0;//TARMAC_DEFAULT_DISTANCE - 10.0;
-
+    public static final double STARTING_DISTANCE = TARMAC_DEFAULT_DISTANCE - 10.0;
```
Comment (prensing) [inline src/main/java/frc/robot/Constants.java:209]: Was hand adjusted at Revere. Please leave as was.

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -130,6 +130,21 @@ private void configureButtonBindings() {
         // JoystickButton farm11 = new JoystickButton(m_farm, 11);
         // farm11.whenPressed(new ResetElevatorEncoder(m_climber));
 
+        // Additional manual shooter position buttons(orange buttons)
+        JoystickButton farm4 = new JoystickButton(m_farm, 4);
+        farm4.whenPressed(new ShooterCommand(m_shooter, m_intake, Constants.TARMAC_DEFAULT_DISTANCE, true));
```
Comment (prensing) [inline src/main/java/frc/robot/RobotContainer.java:135]: What is the use for this? It is the same shot as XBOX_A. Maybe have a shot further inside the Tarmac?

**Review by prensing — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java (hunk 2 above)
Comment (cbf9427) [reply src/main/java/frc/robot/Constants.java:209]: Complete with the misleading comment?

File: src/main/java/frc/robot/RobotContainer.java (hunk 3 above)
Comment (cbf9427) [reply src/main/java/frc/robot/RobotContainer.java:135]: Cognitive consistency. 

I'd be fine if we made this closer, but didn't have shoot numbers I trusted for that. Feel free to change in testing.

**Review by prensing — APPROVED** [review]

**Pull request merged**